src/fides/api/ops/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -304,6 +454,7 @@ def get_historical_consent_reporting(
             PrivacyPreferenceHistory.id,
             PrivacyRequest.id.label("privacy_request_id"),
             PrivacyPreferenceHistory.email.label("user_id"),
```

Comment:
I know this code already existed, but can you tell me more about why email is labeled as the `user_id`? For example, if a privacy center is configured to only accept phone number, would this field be `null`, and would the phone show up under `secondary_user_ids`?

Reply:
Good highlight.  I'd like to update this so email and phone number have their own columns.

 It was like this because `user_id` was a field requested in the consent reporting requirements and it was intended to be the primary identifier, so the email, the phone number, or the fides user device id. But then I started deviating from that because for filtering purposes it's going to be a lot simpler to keep each "type" of data in its own column.  I forgot to go back and update this. 

I intend for secondary_user_ids to be the user ids beyond the "primary id" (mostly ids we get from the browser)

Reply:
Updated!